tests/sentry/api/endpoints/test_group_external_issue_details.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import, print_function
+
+from sentry.models import PlatformExternalIssue
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers import with_feature
+
+
+class GroupExternalIssueDetailsEndpointTest(APITestCase):
+    def setUp(self):
+        self.login_as(user=self.user)
+
+        self.group = self.create_group()
+        self.external_issue = PlatformExternalIssue.objects.create(
+            group_id=self.group.id,
+            service_type='sentry-app',
+            display_name='App#issue-1',
+            web_url='https://example.com/app/issues/1',
+        )
+
+        self.url = u'/api/0/issues/{}/external-issues/{}/'.format(
+            self.group.id,
+            self.external_issue.id,
+        )
+
+    @with_feature('organizations:sentry-apps')
+    def test_deletes_external_issue(self):
+        response = self.client.delete(self.url, format='json')
+
+        assert response.status_code == 204, response.content
+        assert not PlatformExternalIssue.objects.filter(
+            id=self.external_issue.id,
+        ).exists()
+
+    @with_feature('organizations:sentry-apps')
+    def test_handles_non_existing_external_issue(self):
```

Comment:
Is it worth having a test where you attempt to delete an issue for a project/group/organization you're not assigned to?

Reply:
I should think so